classifier/__init__.py

```diff
@@ -856,6 +856,8 @@ def __new__(cls, value):
         if isinstance(value, Lowercased):
             # Nothing to do.
             return value
+        if not isinstance(value, basestring):
+            value = unicode(value)
```

Comment:
This fixes a crash reported by Robert Williams -- I'm not sure how this happened, but probably a Dewey Decimal number was being passed into this constructor.